testing/python/language/test_tilelang_language_reducer_v2.py

```
@@ -1241,5 +1335,49 @@ def kernel(
+    """A reserved dst takes its layout from finalize's verdict, but honoring
+    can be genuinely unsatisfiable: here the consumer loop also reads a
+    fragment annotated to a layout that conflicts with the induced (narrow)
+    verdict, so every free-mode ordering dies on re-validation. The engine
+    must then retry with the universally readable wide fallback (replicated
+    dst) instead of failing with "no available layout found"."""
+    M, K, threads = 16, 64, 256
+
+    @T.prim_func
+    def kernel(
+        A: T.Tensor((M, K), T.float32),
+        O: T.Tensor((M,), T.float32),
```

Ruff (0.16.2)

[error] Ambiguous variable name: `O`

(E741)